packit/upstream.py

```diff
@@ -369,15 +368,10 @@ def get_spec_release(
             string which is meant to be put into a spec file %release field by packit
         """
         original_release_number = self.specfile.expanded_release.split(".", 1)[0]
-        if release_suffix is not None:
-            return (
-                f"{original_release_number}.{release_suffix}"
-                if release_suffix
-                else None
-            )
 
-        if not bump_version:
-            return None
+        if release_suffix is not None:
+            if release_suffix:
+                return f"{original_release_number}.{release_suffix}"
```

Comment:
are both the first and second lines needed? or am I missing something here

Reply:
Yes, because when `release_suffix` is a void string '' should be treated like if it was `None`.
Probably I should make the comparison more explicit like this:
```suggestion
        if release_suffix is not None:
            if release_suffix in not '':
                return f"{original_release_number}.{release_suffix}"
```

Reply:
I understand, so isn't only the second condition enough (`if release_suffix: ...`)?